bin/gen_PKGBUILD.py

```diff
@@ -0,0 +1,44 @@
+import tomli
+
+with open("pyproject.toml", 'rb') as f:
+    pyproject = tomli.load(f)
+
+description = pyproject["project"]["description"]
+version = open('anesthetic/_version.py','r').readlines()[0].split("=")[1].strip().strip("'")
+url = pyproject["project"]["urls"]["Homepage"]
+pyproject["project"]["dependencies"]
+rel=2
+
+
+PKGBUILD = """# Maintainer: Will Handley <wh260@cam.ac.uk> (aur.archlinux.org/account/wjhandley)
+pkgname=python-anesthetic
+_name=${pkgname#python-}
+pkgver=%s
+pkgrel=%s
+pkgdesc="%s"
+arch=(any)
+url="%s"
+license=(MIT)
+groups=()
+depends=(python-numpy python-matplotlib python-scipy python-pandas)
+makedepends=(python-build python-installer)
+provides=()
+conflicts=()
+replaces=()
+backup=()
+options=(!emptydirs)
+install=
+source=("https://files.pythonhosted.org/packages/source/${_name::1}/$_name/$_name-$pkgver.tar.gz")
+sha256sums=(XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX)
+
+build() {
+    cd "$srcdir/$_name-$pkgver"
+    python -m build --wheel --no-isolation
+}
+
+package() {
+    cd "$srcdir/$_name-$pkgver"
+    python -m installer --destdir="$pkgdir" dist/*.whl
+}
+""" % (version, rel, description, url)
```

Comment:
I think this would be far more readable and less error prone with in-line F-strings.

Reply:
Any idea how to do this neatly with an f string that includes `#`?

Reply:
Python only disallows comments inside the `f`-string expressions, so 
```python
>>> """
... ## Hello 
... {20+22}
... """
'\n## Hello \n{20+22}\n'
```

Interprets fine. 

In this case it would look like:

```python
f"""# Maintainer: Will Handley <wh260@cam.ac.uk> (aur.archlinux.org/account/wjhandley)

#... snip
pkgver={version}
pkgrel={rel}
# ... snip
"""
```